BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -3402,10 +3416,10 @@ public void RunTrajectorySimulation()
 
                             var currentTargetDistance = Mathf.Min(maxTargetingRange, (simCurrPos - targetPosition).magnitude);
                             simDeltaTime = Mathf.Clamp(currentTargetDistance / simVelocity.magnitude / 2f, Time.fixedDeltaTime, Time.fixedDeltaTime * BDArmorySettings.BALLISTIC_TRAJECTORY_SIMULATION_MULTIPLIER);
-                            var timeToCPA = AIUtils.ClosestTimeToCPA(targetPosition - simCurrPos, targetVelocity - simVelocity, targetAcceleration - gravity, maxTargetingRange / baseBulletVelocity - simTime); // For aiming, we want the closest approach to refine our aim.
```

Comment:
baseBulletVelocity isn't set for rockets, and thus remains -1, which was causing turrets to be unable to aim at targets outside of the rocket's powered flight range